clinica/iotools/utils/data_handling.py

```diff
@@ -58,11 +59,14 @@ def create_merge_file(
             raise IOError("The path to the CAPS directory is wrong")
 
     if not os.path.isfile(path.join(bids_dir, "participants.tsv")):
-        raise IOError("participants.tsv not found in the specified BIDS directory")
-    participants_df = pd.read_csv(path.join(bids_dir, "participants.tsv"), sep="\t")
+        participants_df = create_participants_df(bids_dir)
+        # raise IOError("participants.tsv not found in the specified BIDS directory")
```

Comment:
Since the BIDS specification does not make `participants.tsv` obligatory, I think this part is a welcome change. However, I think it is redundant with the call to `get_subject_session_list` below it, which will also traverse the BIDS / CAPS tree to list all participants (and respective sessions).

What do you think?

Reply:
You are right, I added this function because the code was already using  `participants.tsv` for extracting the subject list. Updated.